### DISCUSSION TITLE
Checklist Markdown Shortcut

### DISCUSSION THREAD

**Discussion by antran22** [outline/outline#5559] [Ideas]
A common feature with Markdown that I found myself doing a lot is typing the square bracket pair `[]` to open a Checkbox. This can be a quality of life improvement for Outline

**Comment by tommoor** [comment]
This is already the case

<img width="346" alt="image" src="https://github.com/outline/outline/assets/380914/58949313-aad0-45a0-8d6f-1a80ab61e7c0">
